.travis.yml

```diff
@@ -11,6 +11,14 @@ script:
 
 jobs:
   include:
+  - stage: best practices
+    rust: stable
+    install:
+      - rustup component add rustfmt
+      - rustup component add clippy
+    script:
+      - rustfmt --check src/lib.rs
```

Comment:
Is there a better way to do this? Apparently `cargo fmt` doesn't support the `--check` flag, so I have to use `rustfmt` directly and specify the files to check.